executor/infoschema_reader.go

```diff
@@ -17,16 +17,23 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	"io/ioutil"
+	"net/http"
 	"sort"
+	"strconv"
 	"strings"
 	"sync"
 	"time"
 
+	"go.etcd.io/etcd/clientv3"
+
```

Comment:
remove this blank line and reformat code.

Reply:
Fixed.